src/main/java/com/user/management/appl/facade/user/impl/UserFacadeImpl.java

```diff
@@ -81,4 +81,16 @@ public User getUsername(String username) {
     public User updatePassword(User user) {
         return userDao.updatePassword(user);
     }
+
+    @Override
+    public boolean forgotPassword(String username, String securityQuestionAnswer, String newPassword) {
```

Comment:
this implies that all usernames are unique, is there a way to check it?

Reply:
I haven't thought of a way to check it po, because I only have one data in the database

Reply:
But yes, i implies that all username registered is unique po

Reply:
can you try registering two users with the same usernames and test what happens with this?

Reply:
![test](https://github.com/RoCS2024/user-management-system/assets/150970652/eb684b02-39f4-4a6d-9814-d303bee42322)
This is what happened po